simbad/rotsearch/amore_search.py

```diff
@@ -210,16 +217,19 @@ def run(self, models_dir, nproc=2, shres=3.0, pklim=0.5, npic=50,
                 for dat_model, amore_log in zip(dat_models, amore_logs):
                     base = os.path.basename(amore_log)
                     pdb_code = base.replace("amore_", "").replace(".log", "")
-                    RP = simbad.parsers.rotsearch_parser.RotsearchParser(
-                        amore_log
-                    )
-                    score = simbad.rotsearch.amore_score.AmoreRotationScore(
-                        pdb_code, dat_model, RP.alpha, RP.beta, RP.gamma,
-                        RP.cc_f, RP.rf_f, RP.cc_i, RP.cc_p, RP.icp,
-                        RP.cc_f_z_score, RP.cc_p_z_score, RP.num_of_rot
-                    )
-                    if RP.cc_f_z_score:
-                        results += [score]
+                    try:
+                        RP = simbad.parsers.rotsearch_parser.RotsearchParser(
```

Comment:
Please use something more descriptive than `RP`. 1) Don't use uppercase variables for non-constants; 2) this should by `rotsearch_parser`.